### PR TITLE
gtkwave: update to 3.3.107

### PR DESCRIPTION
**Pull request by umarcor**
This is a minor update, with some enhancements to the PKGBUILD file.

### CODE REVIEW

File: mingw-w64-gtkwave/PKGBUILD [hunk 1]
```diff
@@ -43,6 +42,10 @@ build() {
   make
 }
 
+post_install() {
+  ${MINGW_CHOST}/bin/gtk-update-icon-cache -q -t -f ${MINGW_CHOST}/share/icons/hicolor
```
Comment (lazka) [inline mingw-w64-gtkwave/PKGBUILD:46]: What does this do? I can't find it in the docs.

Comment (umarcor) [reply mingw-w64-gtkwave/PKGBUILD:46]: According to https://wiki.archlinux.org/index.php/PKGBUILD#install:

> post_install — The script is run right after files are extracted. One argument is passed: new package version.

Comment (Alexpux) [reply mingw-w64-gtkwave/PKGBUILD:46]: @umarcor port install scripts are separate scripts. See other packages

Comment (umarcor) [reply mingw-w64-gtkwave/PKGBUILD:46]: @Alexpux, which would be the adequate stage in PKGBUILD to execute that command, without requiring separate scripts?

On the other hand, I compared the artifact of this PR with the one from https://packages.msys2.org/package/mingw-w64-x86_64-gtkwave?repo=mingw64, and there is not difference. Hence, either `post_install` is being used, or the command is useless anyway.

Comment (Alexpux) [reply mingw-w64-gtkwave/PKGBUILD:46]: @umarcor look into https://github.com/msys2/MINGW-packages/tree/master/mingw-w64-shared-mime-info for example

**Comment by Alexpux** [comment]
@umarcor you cammot merge install scripts into PKGBUILD, they must be separate

**Comment by Alexpux** [comment]
@umarcor back scripts as they was

**Pull request merged**